### PR TITLE
docs: translate specification

### PR DESCRIPTION
**Pull request by benjycui**
The design principles of Ant Design.
#1471

### CODE REVIEW

**Comment by benjycui** [comment]
I am not sure if you are interested in reviewing those documentations. @waywardmonkeys @michshat @icukaegbu

File: docs/spec/contrast.en-US.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+In order to help user make a quick operation (something like the form，modal), a more important operation or a operation with higher frequency would be emphasized. 
+
+
+> Notes: ways of emphasizing are not just to intensify the key item. It could also weaken the other items. 
+
+
+<br>
+
+<img class="preview-img" align="right" alt="Example of ignoring the primary and secondary sequence" decription="『通过』和『驳回』都使用次按钮，系统保持中立。" src="https://os.alipayobjects.com/rmsportal/xskurfmyKPumFSv.png">
```
Comment (afc163) [inline docs/spec/contrast.en-US.md]: decription is still in Chinese...

File: docs/spec/reaction.en-US.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+<img class="preview-img" align="right" alt="example of Live Preview" description="Depending on the type of the input from the user, the system provides instant feedback on password strength and validation." src="https://os.alipayobjects.com/rmsportal/jecYhRgfbHleGDJ.png">
+
+Live Preview: A Live Preview gives the users a glimpse beforehand of how the application will interpret their input once submitted.
+
+>Note: An ounce of prevention is worth a pound of cure. Use Live Previews to prevent errors.
+
+<br>
+
+Progressive Disclosure: When users are faced with a series of steps, it is often best to provide hints only when they are needed, instead of cluttering the interface by displaying all the hints at once. Learn more cases on [Stay on the Page/Progressive Disclosure](/docs/spec/stay#流程处理)。
```
Comment (afc163) [inline docs/spec/reaction.en-US.md]: `流程处理`

File: docs/spec/stay.en-US.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+---
+
+## Virtual Pages
+
+In the process of interaction design, Overlays allow you to bring additional interactions or content in a layer above the cur- rent page. Inlays allow you to do this within the page itself. However, another powerful approach to keeping users engaged on the current page is to create a virtual page. That is to say, we create the illusion of a larger virtual page. 
+
+Virtual Scrolling and Pagination, more on [Patterns／Lists／Show Long Lists](../pattern/list#显示长列表)
+
+Carousel, more on [Patterns／Lists／Show Images](../pattern/list#显示图片)
```
Comment (afc163) [inline docs/spec/stay.en-US.md]: Hash should be translated too.

**Comment by icukaegbu** [comment]
@benjycui will look into it later today, trying to work through a deadline for the next 4 hours. Cheers

**Comment by waywardmonkeys** [comment]
I am interested... Just traveling in Indonesia this week.

**Comment by benjycui** [comment]
ping~

**Comment by afc163** [comment]
@icukaegbu @waywardmonkeys We need your help~

**Comment by icukaegbu** [comment]
@afc163, @benjycui,  I'm here, apologies for the long silence, was tied down. Where do I start? Can you point me in the direction to go?

**Comment by benjycui** [comment]
@icukaegbu just read those translations, and point out some Chinglish which should be fixed. For we are not native speakers, and may make some silly mistakes. It's unnecessary and impossible to be perfect. Thanks :-)

File: docs/spec/stay.en-US.md (hunk 3 above)
Comment (benjycui) [inline docs/spec/stay.en-US.md]: /docs/pattern/* have not been translated, so keep it until pattern is translated.

**Comment by benjycui** [comment]
Merge now, anyone can create a PR to improve these translations anytime.

**Pull request merged**